src/scss/sections/brands/brands.scss

```diff
@@ -14,32 +14,33 @@
             padding-top: 0;
         }
     }
+}
 
-    .brand {
-        display: flex;
-        justify-content: center;
-        width: calc( 100% / 2);
-        margin-bottom: 20px;
-        padding: 0 28px;
+.brand {
+    display: flex;
+    justify-content: center;
+    width: calc( 100% / 2);
+    margin-bottom: 20px;
+    padding: 0 28px;
 
         a:hover {
-			svg path {
-				fill: #6188ff;
+            svg path {
+                transition: $tr;
```

Comment:
Why is it here? Where you need to place transition? Why only `$tr`? What about `fill ease-out`?